src/cli.ts

```diff
@@ -85,14 +85,27 @@ const detect = defineCommand({
       type: "string",
       description: "Current working directory",
     },
+    warn: {
+      type: "boolean",
+      description: "Log warnings",
+      default: true,
+    },
   },
   run: async ({ args }) => {
     const cwd = resolve(args.cwd || ".");
     const packageManager = await detectPackageManager(cwd);
+
+    if (args.warn && packageManager?.warnings) {
```

Comment:
I think warn by default in cli is fine. If someone wants to suppress warn/errors can do `2>dev/null`